atmo/clusters/models.py

```diff
@@ -150,6 +152,9 @@ def provisioner(self):
     def get_absolute_url(self):
         return reverse('clusters-detail', kwargs={'id': self.id})
 
+    def get_full_url(self):
+        return urljoin(settings.SITE_URL, self.get_absolute_url())
```

Comment:
I'm rather safe than sorry about stuff like this, in case the `SITE_URL` stops having a trailing slash for example, this is a safe bet. This also deduplicates slashes, which string.join wouldn't.